Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import Assigments.Array;\n\npublic class Main {\n    public static void main(String[] args) {\n        int[]array = {30, 20 ,10, 60 ,80 ,1000};\n        System.out.println(Array.getMaximum(array));\n        System.out.println(Array.getMinimum(array));\n\n\n\n//        int[] numbers = {1, 2, 3, 4, 5};\n//                for (int number : numbers){\n//                    System.out.println(number);\n//                }\n\n    }\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
--- a/src/Main.java	(revision c1a8b2a5c82509f7b19bab2bb6f5c5d945f14bb0)
+++ b/src/Main.java	(date 1683723563308)
@@ -13,5 +13,7 @@
 //                    System.out.println(number);
 //                }
 
+
+
     }
 }
\ No newline at end of file
Index: src/JavaPassByValue.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/JavaPassByValue.java b/src/JavaPassByValue.java
new file mode 100644
--- /dev/null	(date 1683724614149)
+++ b/src/JavaPassByValue.java	(date 1683724614149)
@@ -0,0 +1,12 @@
+public class JavaPassByValue {
+    public static void main(String[] args) {
+        int number = 10;
+        modify(number);
+        System.out.println(number);
+    }
+
+    public static void modify(int number){
+        number = 40;
+        System.out.println(number);
+    }
+}
Index: src/PassByReference.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/PassByReference.java b/src/PassByReference.java
new file mode 100644
--- /dev/null	(date 1683725123716)
+++ b/src/PassByReference.java	(date 1683725123716)
@@ -0,0 +1,17 @@
+import java.util.Arrays;
+
+public class PassByReference {
+    public static void main(String[] args) {
+        int[] number = {1, 2, 3, 4, 5};
+        modify(number);
+        System.out.println(Arrays.toString(number));
+    }
+
+    public static void modify(int[]arr){
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = arr[i] * arr[i];
+        }
+        System.out.println(Arrays.toString(arr));
+
+    }
+}
